### DISCUSSION TITLE
GO-GOslim file link broken

### DISCUSSION THREAD

**Discussion by yaaminiv** [RobertsLab/resources#1534] [Q&A]
I'm using the workflow in [this Jupyter notebook](https://github.com/RobertsLab/code/blob/master/10-blast-2-slim.ipynb) to map gene IDs to GOSlim. However, the link for the GO-GOslim file is broken: http://owl.fish.washington.edu/halfshell/bu-alanine-wd/17-07-20/GO-GOslim.sorted

Where is the new location of this file? I found it in one of my old repos but don't know where the canonical version is.

**Comment by sr320** [comment]
that one is gone  - but this should be same https://gannet.fish.washington.edu/spartina/project-oyster-oa/Haws/GO-annotation/GO-GOslim.sorted

**Comment by yaaminiv** [comment]
@sr320 @kubu4 How was this file originally made? I'd like to use it for a project and include some sort of description

**Reply by sr320** [reply]
from Gavery and Roberts 2012

>Associated GO terms (Gene Ontology database: http://www.geneontology.org/) were used to categorize genes into parent categories and were assigned a functional group based on the MGI GO Slim database (URL: http://www.informatics.jax.org/). The MGI GO Slim terms for ‘other biological processes’ and ‘other metabolic processes’ are not included in this analysis.
